AWS-CREDENTIALS.md

```diff
@@ -79,9 +79,6 @@ let auto_refreshing_provider = rusoto_credential::AutoRefreshingProvider::new(pr
 
 #### Local integration testing of IAM credentials
 
-Edit the relevant `address`/IP locations in [credential/src/container.rs](credential/src/container.rs) and [credential/src/instance_metadata.rs](credential/src/instance_metadata.rs).
-For local testing, you can use [moe](https://github.com/matthewkmayer/moe) and set the string to this:
-
-```rust,ignore
-let address: String = "http://localhost:8080/latest/meta-data/iam/security-credentials".to_owned();
-```
+Local testing of IAM credentials can be done using [credential_service_mock](rusoto/credential_service_mock) which runs as a local [Warp](https://crates.io/crates/warp) server.
+ 
+Before running the mock server, change the values of the `AWS_CREDENTIALS_PROVIDER_IP` constants in [credential/src/container.rs](rusoto/credential/src/container.rs) and [credential/src/instance_metadata.rs](rusoto/credential/src/instance_metadata.rs) to either `localhost:8080` or `127.0.0.1:8080`
```

Comment:
Can we change this to reflect how the test is implemented? https://github.com/rusoto/rusoto/blob/master/rusoto/credential/tests/instance-profile-test.rs#L15 shows how to override the endpoint without having to modify the credentials source code. 👍

Reply:
Hm, I'm not sure of how that should be worded. I may not be a good person for this issue after all. I ran the tests using the makefile but some tests failed and I'm not sure why. I should spend some more time learning how the credentials testing works before continuing. I apologize for opening up the pull request prematurely.

Reply:
No worries, you're doing fine. 😄 

Since we have recent changes in the credential providers to allow overriding of source locations, we could remove line 84 entirely and replace it with a sentence saying "see the integration test at https://github.com/rusoto/rusoto/blob/master/rusoto/credential/tests/instance-profile-test.rs for a working example of local testing."

How's that sound? 😄

Reply:
That makes sense, thank you very much. Sometimes the imposter syndrome locks up the brain and all. Again, thank you. I'll make the change and have it ready shortly